docs/user/metrics/labeled_booleans.md

```diff
@@ -130,6 +130,39 @@ Assert.Equal(0, Accessibility.features.TestGetNumRecordedErrors(ErrorType.Invali
 
 </div>
 
+<div data-lang="Rust" class="tab">
+
+```rust
+use glean_metrics;
+
+accessibility::features.get("screen_reader").set(is_screen_reader_enabled());
```

Comment:
Should we implement the `Index` [trait](https://doc.rust-lang.org/std/ops/trait.Index.html) so we could say:

```rust
accessibility::features["screen_reader"].set(is_screen_reader_enabled());
```

This would be more in line with other language bindings.

I assume (but don't know) that Traits can require other Traits to make this happen...

Reply:
I think that's a really good idea. @badboy any reason why this wasn't implemented in m-c?

Reply:
`Index` returns a reference to an existing object. As we don't have an existing object when the user just calls it we can't create a reference out of thin air, thus we can't implement `Index` for this.

Reply:
I see. Thanks for explaining.